pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -191,11 +207,8 @@ func (s *completed) Execute(msg dispatcher.DIDCommMsg) (state, error) {
 		// send ACK
 		return &noOp{}, nil
 	case connectionAck:
-		if msg.Outbound {
-			// send ACK
-			return &noOp{}, nil
-		}
-		// save did-exchange connection
+		// if msg.Outbound send ACK
```

Comment:
I just see a re-ordering or the comments here. No real need for it though.

Reply:
I guess the comment is a bit confusing.

(we are inside a case where there is a comment of if ... otherwise; and no corresponding if/else statement.)

Is the "otherwise" referring to the default case?

Reply:
yes, `otherwise` is equal to the default, as you can see below there is no need to return twice the same response
I just left those comments because it can be helpful (to understand that there is two behavior) but if it redundant or outdated better to remove it
```
if msg.Outbound {
    // send ACK
   return &noOp{}, nil
}
// save did-exchange connection
return &noOp{}, nil
```